### ISSUE TITLE
Video Feature Extraction

### ISSUE THREAD

**Issue by Alen-T** [hazeld/action-modifiers#1]
Hi, Can you provide code to extract I3D (RGB and Flow) features? Thank you very much!

**Comment by Stomachache007** [comment]
Yeah, really looking forward for your extract I3D features. Thanks!
